kbhff/test/login_test.py

```diff
@@ -130,7 +130,19 @@ def test_unverifiedUserWithPasswordFirstLoginDuplicateVerificationTab(driver, gm
     assert_username_prefilled(user["email"], driver)
     assert_text_on_page("Din konto er allerede aktiveret!", driver)
 
-#Test specification number 7 is a duplicate of number 5
+#Test specification number 7, 
+#contrary to specification this is test 5 with signup via medlemshjaelp
+def test_unverifiedUserWithoutPasswordFirstLoginIncorrectCode(driver, unverified_user_via_medlemshjaelp):

```

Comment:
I think `test_unverifiedUserWithoutPassword_firstLogin_incorrectCode` is clearer